tools/debugger/tests/do_command.expected

```diff
@@ -0,0 +1,11 @@
+Connecting to: localhost:6501
+Connection created!!!
+Stopped at tools/debugger/tests/do_command.js:2
+Breakpoint 1 at tools/debugger/tests/do_command.js:2
+0Print: 0
+Stopped at breakpoint:1 tools/debugger/tests/do_command.js:2
```

Comment:
Some initial part of expected result seems incorrect.
According to your predefined command argument, `e i;b do_command.js:2;c;c;c`,
result order should be shown as follow
1) result of evaluation `i`
2) add break point at line 2
3) continue the execution ...

But the actual result is,
1) add break point as line 2
2) result of evaluation `i`
3) continue the execution ...

Reply:
(One more thing)
At the start of debugger, it first stoped at `tools/debugger/tests/do_command.js:2`
But IMO it should be stop at line 1 not line 2.